charts/tekton-pipelines/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.29-bump
+version: 0.1.31-dev.1
```

Comment:
@roman-mightydevops pls merge this PR into yours
https://github.com/saritasa-nest/saritasa-devops-helm-charts/pull/55
before merging into main, as Sergey has a lower version 0.1.30 that uses yamlfix implementation to beautify apps/ yaml manifests.